### PR TITLE
INTEGRATED-1221 - Added poll with configuration + added README.md, gi…

### PR DESCRIPTION
**Pull request by pappi1987**
…tignore

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?php
```
Comment (joenivl) [inline DependencyInjection/Configuration.php]: File can be removed

File: DependencyInjection/IntegratedPollExtension.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?php
```
Comment (joenivl) [inline DependencyInjection/IntegratedPollExtension.php]: File can be removed

File: Document/Poll.php [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+*
+* (c) e-Active B.V. <integrated@e-active.nl>
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+namespace Integrated\Bundle\PollBundle\Document;
+
+use Doctrine\ODM\MongoDB\Mapping\Annotations as ODM;
```
Comment (joenivl) [inline Document/Poll.php]: Mapping should be XML, so remove the ODM annotations.

**Review by joenivl — COMMENTED** [review]
.travis file is missing. Can you add one with the specific checks so this project can be attached in Jenkins

File: DependencyInjection/Configuration.php (hunk 1 above)
Comment (pappi1987) [reply DependencyInjection/Configuration.php]: File removed.

File: DependencyInjection/IntegratedPollExtension.php (hunk 2 above)
Comment (pappi1987) [reply DependencyInjection/IntegratedPollExtension.php]: File removed.

File: Document/Poll.php (hunk 3 above)
Comment (pappi1987) [reply Document/Poll.php]: Removed ODM annotations

**Pull request merged**